src/interfaces.ts

```diff
@@ -37,3 +41,12 @@ export interface OtherInfo {
   javaNotInstalled?: boolean;
   nonDefaultConfigName?: string;
 }
+
+export interface AndroidResultSetup {
+  status: boolean;
+  setup: boolean;
+}
```

Comment:
No need for writing this, can be taken directly from `nightwatch-mobile-helper`.